types/git-http-backend/tsconfig.json

```
@@ -0,0 +1,24 @@
+{
+  "compilerOptions": {
+      "module": "commonjs",
+      "lib": [
+          "es6",
+          "dom"
```

```suggestion
          "es6"
```
why 'lib.dom' is required here?

---

i just copy pasted from another type definition and i forgot to remove